From f41b1dca7c5fe9be098576e976d2227a388cffe0 Mon Sep 17 00:00:00 2001
From: abhiram <abhithegamer2019@gmail.com>
Date: Mon, 24 May 2021 17:34:31 +0530
Subject: [PATCH] Disable Bungee updator


diff --git a/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java b/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java
index 6cd4769e..66a2c941 100644
--- a/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java
+++ b/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java
@@ -57,23 +57,7 @@ public class BungeeCordLauncher
             System.out.println( BungeeCord.class.getPackage().getImplementationVersion() );
             return;
         }
-
-        if ( BungeeCord.class.getPackage().getSpecificationVersion() != null && System.getProperty( "IReallyKnowWhatIAmDoingISwear" ) == null )
-        {
-            Date buildDate = new SimpleDateFormat( "yyyyMMdd" ).parse( BungeeCord.class.getPackage().getSpecificationVersion() );
-
-            Calendar deadline = Calendar.getInstance();
-            deadline.add( Calendar.WEEK_OF_YEAR, -8 );
-            if ( buildDate.before( deadline.getTime() ) )
-            {
-                System.err.println( "*** Hey! This build is potentially outdated :( ***" );
-                System.err.println( "*** Please check for a new build from https://papermc.io/downloads ***" );
-                System.err.println( "*** Should this build be outdated, you will get NO support for it. ***" );
-                System.err.println( "*** Server will start in 10 seconds ***" );
-                Thread.sleep( TimeUnit.SECONDS.toMillis( 10 ) );
-            }
-        }
-
+        
         BungeeCord bungee = new BungeeCord();
         ProxyServer.setInstance( bungee );
         // RedCraft-Proxy start - Rebrand Travertine
-- 
2.26.2.windows.1

